modules/core/src/main/java/org/terrier/matching/models/WeightingModel.java

```diff
@@ -171,14 +169,14 @@ public void setRequest(Request _rq) {
 	 * @param _c the term frequency normalisation parameter value.
 	 */
 	public void setParameter(double _c) {
-		this.c = _c;
+		
```

Comment:
should the documentation be updated to indicate that getParameter and setParameter are no longer supported?